### PR TITLE
Let engine handle defaulting the scaling strategy

### PR DESCRIPTION
**Pull request by mattkelly**
#### What does this pull request accomplish?

Let engine handle defaulting the scaling strategy

 #### What issue(s) does this fix?
* None

 #### Additional Considerations

 ### Testing

 ```
 2018-12-09T17:15:19.000-0800    INFO    Event(v1.ObjectReference{Kind:"AutoscalingGroup", Namespace:"", Name:"tester", UID:"b95ea3a8-f99f-11e8-8a80-0697bcac5b18", APIVersion:"cerebral.containership.io/v1alpha1", ResourceVersion:"428809", FieldPath:""}): type: 'Normal' reason: 'ScaledDown' scaled down to 3 nodes using strategy ""
 ```

 Note that `strategy ""` is printed. I could make this prettier but it seems not worth the effort IMO.

 ### Dependencies
* None

### CODE REVIEW

File: pkg/autoscalingengine/containership/containership_test.go [hunk 1]
```diff
@@ -45,10 +45,7 @@ func TestSetTargetNodeCount(t *testing.T) {
 	assert.Error(t, err, "Testing that an error is returned if there is a request to scale below 0")
 	assert.False(t, result)
 
-	nodePoolLabel := map[string]string{
-		nodePoolIDLabelKey: "node-pool-uuid",
-	}
-	result, err = c.SetTargetNodeCount(nodePoolLabel, 2, "")
-	assert.Error(t, err, "Testing that an error is returned if no strategy is specified")
-	assert.False(t, result)
+	// TODO add a test checking that providing an empty string for strategy is ok
```
Comment (mattkelly) [inline pkg/autoscalingengine/containership/containership_test.go]: Sorry @ashleyschuett  :|
